src/export/InstanceExporter.ts

```diff
@@ -26,6 +27,37 @@ export class InstanceExporter implements Fishable {
     private readonly fisher: Fishable
   ) {}
 
+  /**
+   * Checks if there exist Instances of Custom Resources, which are not supported
+   * by the IG Publisher and are therefore not included in the IG.
+   */
+  private warnOnInstanceOfCustomResource(): void {
+    const instancesOfCustomResource = this.pkg.instances.filter(instance =>
+      this.pkg.resources.some(r => r.type === instance.resourceType)
+    );
+    if (instancesOfCustomResource.length) {
+      // logger.warn color
+      const clr = chalk.rgb(179, 98, 0);
+      const maxLength = 61;
+      const instancesOfCustomResourceLogs = instancesOfCustomResource.map(r => {
+        const printableName = r.id.length > maxLength ? r.id.slice(0, maxLength - 3) + '...' : r.id;
+        return clr('│') + padEnd(` - ${printableName}`, 65) + clr('│');
+      });
+
+      // prettier-ignore
+      const message = [
+          clr('\n╭─────────────────────────────────────────────────────────────────') + '' + clr('╮'),
+            clr('│') + ' Detected the following instance(s) of custom resources:         ' + clr('│'),
+            ...instancesOfCustomResourceLogs,
+            clr('│') + '                                                                 ' + clr('│'),
+            clr('│') + ' The IG Publisher does not support custom resources, these       ' + clr('│'),
+            clr('│') + ' instances will not be included in the IG.                       ' + clr('│'),
+            clr('╰─────────────────────────────────────────────────────────────────') + '' + clr('╯\n')
```

Comment:
I like the fancy box!

One thing I'm not sure about is if we want to align the language we use here with the warning we give when we don't add custom resources. In that box, we refer to the resources we filter out as "non-conformant Resource definitions" and we just say they are not conformant to the FHIR specification. This makes me wonder if we should avoid referring to these as "custom resources" here and avoid referring directly to the IG publisher.

Before we workshop the language too much, this might be something we want to double check with @cmoesel on.

Reply:
Another suggestion for the language here, I think we should explicitly say that the instances will not be included in the IG _resource_, rather that the IG itself. I think the warning as it currently exist implies that SUSHI doesn't include the instance in the `fsh-generated` folder at all, which it does.

Reply:
Done.